### DISCUSSION TITLE
WARNING: Unable to resolve crossref @fig-bau-formen

### DISCUSSION THREAD

**Discussion by MichelDSchmid** [quarto-dev/quarto-cli#5568] [Q&A] [answered]
Hi all

I have a, hopefully, small problem with cross references in a book project.

```
![NTC-Bauformen](images/ntc_bauformen.png){#fig-bau-formen width=9.5cm}
NTCs gibt es in verschiedensten Bauformen wie in @fig-bau-formen ersichtlich ist. Diese Bauformen stellen lediglich eine Auswahl der zur Verfügung stehenden Bauformen dar.
```

I'm always getting the error message:
WARNING: Unable to resolve crossref @fig-bau-formen
and of course I can't see a cross reference in the document, indepentend if it's in html or pdf..

I coudn't found something in the docs and I'm using quarto version 1.3.353

Thanks in advanced
Mike

**Comment by dragonstyle** [comment, marked as answer]
Syntactically, in order for the image to be considered a 'figure', it needs to be a block (e.g. on it's own line):

```markdown
![NTC-Bauformen](images/ntc_bauformen.png){#fig-bau-formen width=9.5cm}

NTCs gibt es in verschiedensten Bauformen wie in @fig-bau-formen ersichtlich ist. Diese Bauformen stellen lediglich eine Auswahl der zur Verfügung stehenden Bauformen dar.
```

Without the blank line between the image and the text, the two are placed together in a single paragraph and the image is treated as in inline image rather than a figure.

HTH!

**Reply by cscheid** [reply]
This feels like a good candidate for our future `lint` feature.

Tracking on #5571.
